/storage_path:
  ros__parameters:
    path: './src/camera_package/video/'
